package/resiot/control/control

```diff
@@ -0,0 +1,6 @@
+Package: helium_gateway
+Version: ${CARGO_MAKE_CRATE_VERSION}
+Architecture: armv7ahf-vfp
```

Comment:
This should probably be ${CARGO_MAKE_PROFILE} since it now appears to point at slightly different architecture?

Reply:
Had to pick one from our opkg architecture list as the replaced variable does not appear in the default list. I assumed editing it here was faster than reflashing all the gateways or manually editing opkg configuration settings on the gateways before installing

Reply:
ugh.. you'd think that the architecture would match the build arch target.. oh well. if it works for ya